packages/suite/src/components/suite/HiddenPlaceholder.tsx

```diff
@@ -7,6 +7,11 @@ import { RedactNumbersContext } from '@suite-common/wallet-utils';
 import { useSelector } from 'src/hooks/suite';
 import { selectIsDiscreteModeActive } from 'src/reducers/wallet/settingsReducer';
 
+type MouseCoords = { clientX: number; clientY: number };
+
+const PIXELS_TOLERANCE = 0.5;
+const approxEqual = (a: number = NaN, b: number = NaN) => Math.abs(a - b) < PIXELS_TOLERANCE;
```

Comment:
I used `Nan` so that _either_ `a` or `b` undefined returns `false`. Do you think it warrants a comment?